portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,17 @@
+package com.google.sps.data;
+
+/** A user-inputted comment. */
+public final class Comment {
+
+  private final long id;
+  private final String name;
+  private final String body;
+  private final long timestamp;
+
+  public Comment(long id, String name, String body, long timestamp) {
```

Comment:
You don't need to implement this becuase it's quite wordy and not worth doing for a simple application, but for your education, usually this type of thing would be well served with a "Builder" pattern. 

See:
https://refactoring.guru/design-patterns/builder/java/example